scripts/upgrade

```diff
@@ -37,6 +37,9 @@ fi
 
 if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
+	# purge the "bundles" folder to avoid pointlessly taking up space at each update
+	ynh_secure_remove --file="$install_dir/bundles/*"
```

Comment:
well `*` don't work on helper. But I think you can just remove `bundles` as we setup source after that.

Reply:
yes we could